README.md

```diff
@@ -26,12 +28,18 @@ Alternatively you can import this module in your frontend build chain.
 #### `lib.typoscript`
 ```
 lib.tsExample = TEXT
-lib.tsExample.typolink.htmlSpecialChars = 1
-lib.tsExample.typolink.parameter.data = page:uid
-lib.tsExample.typolink.topwire.type = plugin
-lib.tsExample.typolink.topwire.extensionName = TopwireExamples
-lib.tsExample.typolink.topwire.pluginName = Json
-lib.tsExample.typolink.returnLast = url
+lib.tsExample {
+    typolink {
+        parameter.data = page:uid
+        topwire {
+            type = plugin
+            extensionName = TopwireExamples
+            pluginName = Json
+        }
+        returnLast = url
+        htmlSpecialChars = 1
```

Comment:
Notice that this an error in the original snippet already: `htmlSpecialChars` is an option of `stdWrap` / `TEXT`, not of `typolink`. This should be fixed separately.

Reply:
ack

Reply:
https://github.com/topwire/topwire/pull/32